CHANGELOG.md

```diff
@@ -2,7 +2,15 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [Unreleased]
+### Added
+
+### Fixed
+- Replaced deprecated 'form' type by FormType class
+
+
```

Comment:
@greg0ire so I must have misunderstood the CONTRIBUTING.md instructions. I'll revert asap.